script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -554,14 +562,15 @@ The Dead that Arcueid took down earlier was nothing more than a 'servant',
 [sha:c06cc98ccf51f55304c457ea093c167e5e796138]{
 -- Page 552, Offset 17952.
 -- Ⅰ階梯なら自動人形にすぎないけど、Ⅲ階梯なら脳は生きているし。彼等は自分がどういう生き物になったのかを理解した上で吸血鬼に従っている。自分から進んで人間たちを殺して、親基に血を送っているのよ」
-A Rank I Dead is nothing more than an %{i}automaton%{/i}, but a Rank III still has a functioning brain.
+// no need for italic on automaton, not sure
+Rank Is are nothing more than automatons, but Rank IIIs still have functioning brains.
  They are able to comprehend what kind of creature they've become, yet continue to serve.
- They make the conscious choice to kill humans, and send the blood to their parent vampire."
+ They make the conscious choice to kill humans, and send the blood to their vampire sire."
 }
 [sha:d4599fd7253c62c85bfa3db269957db5b05602e1]{
 -- Page 553, Offset 17953.
 -- 「……いや、それは違うだろ。
-"...No, that can't be right.
+"Wait... that seems off.
```

Comment:
Hm, I feel like `That can't be right` isn't _that_ certain of a statement, it's used fairly often as a 'probing' rather than a 'decreeing' phrase. 'seems off' just feels out of whack with the following sentence